sources/packages/api/src/cra-integration/cra-personal-verification.service.ts

```diff
@@ -106,7 +106,7 @@ export class CRAPersonalVerificationService {
     // Executes the processing of each file in parallel.
     const filesProcess = files.map((file) => this.processResponse(file));
     // Waits for all the parallel processes to be finished.
-    return await Promise.all(filesProcess);
+    return Promise.all(filesProcess);
```

Comment:
Recommendation from Sonar Cloud.

Reply:
oh..but await is need for async right.Okay

Reply:
Yes, it is, but not at this point. The consumer of this method will await this.